### ISSUE TITLE
ERROR:  could not access file "$libdir/plpgsql": No such file or directory

### ISSUE THREAD

**Issue by croxey** [PostgresApp/PostgresApp#87]
Great App for development!  I'm pumped to use it, but I am getting the following error when trying to create a stored procedure.  

```
#CREATE FUNCTION
#...
#END;
#$$ LANGUAGE plpgsql;
ERROR:  could not access file "$libdir/plpgsql": No such file or directory
```

Any idea what might be wrong?  Or how I can fix it?

I checked...

```
$ pg_config --pkglibdir
/Applications/Postgres.app/Contents/MacOS/lib

$ pg_config --libdir
/Applications/Postgres.app/Contents/MacOS/lib
```

I've confirmed that plpgsql.so is in that directory.

Should have mentioned that my version is reported as Version 9.2.2.0 (20).  I just downloaded it a few days ago.

**Comment by jcoleman** [comment]
To anyone reading this thread in the future: in my experience (today) the special release containing both 2.0 and 2.1 versions of PostGIS @jakob links to above (9.2.4.4) has the dylib loading issues that were fixed in 9.2.4.3. I recommend downloading 9.2.4.3 and copying over the two 2.0 PostGIS lib files into that app bundle from 9.2.4.3.

**Comment by balnagendra** [comment]
I was stuck with this problem. I am working with PostGres 9.2 CentOS. I was trying to implement TABLEFUNC extension. It would give me this error $libdir/tablefunc not found.

Thanks to VJPR's comments. he left a hint to look into pg_config and PKGLIBDIR variable. Mine was pointing to /usr/lib64/pgsql. I moved tablefunc.so from /usr/pgsql-9.2/lib to /usr/lib64/pgsql. And voila "CREATE EXTENSION TABLEFUNC" worked.

Thanks Team, great help
